index.js

```diff
@@ -27,3 +31,154 @@ for (let i = 0; i < mdlLks.length; i += 1) {
+  button2.classList.add('project-detail__button');
+  // Append
+  container.append(modalHeader, tagsDiv, imgDiv, textDiv);
+  imgDiv.appendChild(mainImg);
+  textDiv.append(leftDiv, rigthDiv);
+  leftDiv.appendChild(description);
+  rigthDiv.append(techDiv, line, buttonsDiv);
+  modalHeader.append(h2, exitButton);
+  tagsDiv.append(tag1, bullet1, tag2, bullet2, tag3);
+  buttonsDiv.append(button1, button2);
+  buttonsDiv.classList.add('project-detail__buttons');
+  button1.innerText = 'See live';
+  button1.appendChild(btnImg1);
+  button2.innerText = 'See source';
+  button2.appendChild(btnImg2);
+}
+
+// To iterate elements in portfolio data
+for (let i = 0; i < data.length; i++) {
+  // Creating elements
+  const portDiv = document.createElement('div');
+  portDiv.classList.add('port1');
+  const portImg = document.createElement('img');
+  const rghtDiv = document.createElement('div');
+  const portTtl1 = document.createElement('h2');
+  portTtl1.classList.add('title');
+  const deetsDiv = document.createElement('div');
+  const span1 = document.createElement('span');
+  span1.classList.add('clnt');
+  const span2 = document.createElement('span');
+  span2.classList.add('deets');
+  const bullet = document.createElement('img');
+  const bullet2 = document.createElement('img');
+  bullet.classList.add('crcl');
+  bullet2.classList.add('crcl');
+  const span3 = document.createElement('span');
+  const description = document.createElement('p');
+  const tchnlgs = document.createElement('ul');
+  tchnlgs.classList.add('tags');
+  // Technologies for loop
+  for (let j = 0; j < data[i].technologies[j]; j++) {
+    const li = document.createElement('li');
+    li.classList.add('tgs');
+    li.innerText = data[i].tags[j];
+    tchnlgs.appendChild(li);
+  }
+  const btnDiv = document.createElement('div');
+  const btn = document.createElement('button');
+  btn.classList.add('prjctbtn');
+  const btnTxt = document.createElement('span');
+  btnTxt.classList.add('btntxt');
+  // Adding content
+  portImg.src = data[i].image;
+  portImg.alt = `${data[i].title} img`;
+  portTtl1.innerText = data[i].title;
+  span1.innerText = data[i].tags[0];
+  span2.innerText = data[i].tags[1];
+  span3.innerText = data[i].tags[2];
+  bullet.src = 'images/circleicn.png';
+  bullet2.src = 'images/circleicn.png';
+  description.innerText = data[i].description;
+  // Append
+  workSectn.appendChild(portDiv);
+  portDiv.appendChild(portImg);
+  portDiv.appendChild(rghtDiv);
+  rghtDiv.appendChild(portTtl1);
+  rghtDiv.appendChild(deetsDiv);
+  deetsDiv.appendChild(span1);
+  deetsDiv.appendChild(bullet);
+  deetsDiv.appendChild(span2);
+  deetsDiv.appendChild(bullet2);
+  deetsDiv.appendChild(span3);
+  rghtDiv.appendChild(description);
+  rghtDiv.appendChild(tchnlgs);
+  portDiv.appendChild(btnDiv);
+  btnDiv.appendChild(btn);
+  btnTxt.innerText = 'See Project';
+  btn.appendChild(btnTxt);
+  btn.addEventListener('click', () => {
+    projectDetails(data[i]);
+  });
```

Comment:
- Your popup modal does not look like the design template. Remember it is important that you try to replicate the Figma design as closely as possible to give confidence to recruiters in your ability to produce exact business requirements 🙏

Your project:
-- Please fix the width of your popup 🙏 
<img width="957" alt="jeje5je5" src="https://user-images.githubusercontent.com/73607512/216141547-e5027bf3-2ee1-48b0-b86b-d38d675ede2f.PNG">

Design template:
<img width="238" alt="ethseths" src="https://user-images.githubusercontent.com/73607512/216141757-34e89e31-aebe-4814-9a23-9d0d5bc812de.PNG">